src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Deployment/AzureADDeploymentSource.cs

```diff
@@ -22,12 +21,12 @@ public async Task ProcessDeploymentStepAsync(DeploymentStep step, DeploymentPlan
             return;
         }
 
-        var settings = await _azureADService.GetSettingsAsync();
+        var azureADSettings = await _azureADService.GetSettingsAsync();
 
-        var obj = new JsonObject { ["name"] = nameof(AzureADSettings) };
-
-        obj.Merge(JObject.FromObject(settings, JOptions.Default));
-
-        result.Steps.Add(obj);
+        result.Steps.Add(new JsonObject
```

Comment:
@hishamco Was it always broken or are we missing something here? Settings step is for importing settings, but this would mean this would have never worked on the past.

Did you test the export and the import to ensure its exporting the correct value and the import is saving the correct value? I would do an import and ensure that the export exports the correct values.

Reply:
It should work as a Microsoft Account, seems this was missing before. Let me test it very quick

Reply:
@MikeAlhayek is works as expected